notebooks/tutorial_RGDR.ipynb

```diff
@@ -70,7 +70,7 @@
    "outputs": [
```

Comment:
<p>As you can see, the xr.DataArray cluster_map first needs to be transformed to integers for plotting.</p><p> </p><p>The function cluster_labels_to_ints copies the values (dtype: strings) and converts the values to dtype: int. This converted array is added as a coordinate (the original cluster_map.values remain strings).</p><p> </p><p>Maybe it would be better if the cluser_map.values are integers by default (making the plotting intuitive) and that the cluster_labels are added as a coordinate.</p><p> </p><p>These cluster_map coordinates can then be used to do the groupby(labels).mean() (https://github.com/AI4S2S/s2spy/blob/8ff64ba23880bab9e24adf1aa268c328e1f79fc6/s2spy/rgdr/rgdr.py#L517-L522). The latter function will need to be slightly adapted to support this.</p><p> </p><p>What do you think?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AI4S2S/s2spy/pull/94/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>I see the point. I agree that in this case integers are better than unicode strings. But I think it is better to look into it and check the code when these labels are generated rather than just add a conversion in the end. So I prefer that we raise an issue for it and deal with it in another PR. Let's merge this PR for now.</p>